lib/Dialect/TritonGPU/Transforms/UpdateMmaForVolta.cpp

```diff
@@ -13,6 +13,7 @@ using triton::DotOp;
 using triton::gpu::ConvertLayoutOp;
 using triton::gpu::DotOperandEncodingAttr;
 using triton::gpu::MmaEncodingAttr;
+using triton::gpu::SharedEncodingAttr;
```

Comment:
This comment is late and inappropriate for this CR. But I don't understand why this pass has to be implemented via the RewritePattern. I don't think it brings any convenience for the logic we need. Seems to me a naive pass (instead of via RewritePattern) is better suitable here.

Reply:
[Here](https://github.com/openai/triton/pull/1048#discussion_r1067767140) is some context.
I do agree a RewritePattern is not appropriate, will refactor it after this PR.